darts/tests/test_4theta.py

```diff
@@ -50,7 +50,7 @@ def test_theta(self):
         fourtheta.fit(series)
         forecast_theta = theta.predict(20)
         forecast_fourtheta = fourtheta.predict(20)
-        self.assertTrue((forecast_theta - forecast_fourtheta <= 1e-12).all()[0])
+        self.assertTrue((forecast_theta - forecast_fourtheta <= 1e-10).all()[0])
```

Comment:
Did you test whether the output of the models is deterministic in this context?

Reply:
You are right, the output is not deterministic, thus explaining the problem with one of the test (the precision error was doubled)